src/Core/Ast/Parser/NikicPhpParser/FileReferenceVisitor.php

```diff
@@ -169,7 +167,14 @@ private function enterFunction(Node\Stmt\Function_ $node): void
             $name = $node->name->toString();
         }
 
-        $this->currentReference = $this->fileReferenceBuilder->newFunction($name, $this->templatesFromDocs($node));
+        $tags = [];
+        $docNodeCrate = $this->getDocNodeCrate($node);
```

Comment:
NIT: can you switch it with the above line? That way it is consistent with the order in `enterClassLike` and therefore more likely to be picked up as duplicate code prime for refactoring.

Reply:
done